src/sentry/runner/commands/devservices.py

```diff
@@ -159,8 +159,11 @@ def exit_handler(*_):
 @click.option("--project", default="sentry")
 @click.option("--exclude", multiple=True, help="Service to ignore and not run. Repeatable option.")
 @click.option("--fast", is_flag=True, default=False, help="Never pull and reuse containers.")
+@click.option(
+    "--skip-only-if", is_flag=True, default=False, help="Skip 'only_if' checks for services"
```

Comment:
I'm having an ESL moment, what is this trying to do/say?

Reply:
We have a attribute in our devservices configuration called `only_if` which will only enable the dev service if the specified function is met.

For `chartcuterie` I have this configured to be only if the `chart-rendering.enabled` option is set, since in development you usually won't need this running. But it's a little difficult to get this configuration set in our test cases early on, so I opted for a simpler solution of providing devservices cli a way to skip that check and just always boot a devservice.